src/project/python.rs

```diff
@@ -179,6 +196,15 @@ mod test {
     use super::*;
     use indexmap::IndexMap;
 
+    #[test]
+    fn test_pypi_to_string() {
+        let pypi = PyPiRequirement::from_str(">=1.0.0,<2.0.0");
+        assert_eq!(
+            pypi.unwrap().to_string(),
+            "version=\">=1.0.0, <2.0.0\", extras=\"\""
```

Comment:
Could you extend this with some more cases

Reply:
I have improved the test scenario, would you like to give me more examples to add as a test? or just that new one is ok?

Reply:
Just noticed this is not correct as the `">=1.0.0,<2.0.0"` should result in `">=1.0.0,<2.0.0"` again without the `version` and `extra` fields. 

Also empty requirements might want to be writen as `*`
And requirements with `extras` without versions should only write `extras = ["extra"]`

Reply:
now that is using the ``Item`` as you have suggested, therefore that is completely different now, everything is being handled by that

Reply:
I'm unsure what is going on there, i've been testing this pr and it just works in the cli so forget this.